tests/test_api_result.py

```diff
@@ -42,15 +42,15 @@
 )
 from openeogeotrellis.backend import JOB_METADATA_FILENAME
 from openeogeotrellis.job_registry import ZkJobRegistry
+from openeogeotrellis.layercatalog import get_layer_catalog
 from openeogeotrellis.testing import random_name, KazooClientMock, config_overrides
 from openeogeotrellis.utils import (
     UtcNowClock,
     drop_empty_from_aggregate_polygon_result,
     get_jvm,
     is_package_available,
 )
-from openeogeotrellis.vault import OAuthCredentials
-from .data import get_test_data_file
+from tests.data import get_test_data_file
```

Comment:
I'm not sure this is an improvement over the original `from .data import get_test_data_file` (which is also not ideal by the way). This import makes it look like there is some top-level `tests` package in the venv while the relative import makes it more clear it is just from a sibling file.

Is there a particular reason you changed this?